app/views/partials/foyer/individu-form.html

```diff
@@ -162,23 +162,16 @@
       <span class="help-block">Si vous n’avez jamais eu de contrat de travail, laissez ce champ vide.</span>
     </div>
 
-    <div class="form-group" ng-if="individu.lastJobEndDate.year()">
```

Comment:
Yes: the `<yes-no-question>` directive root is a `<fieldset class="form-group">`, so there's no need to repeat it on the outside of the component.